packages/next/src/template-renderer/index.tsx

```diff
@@ -53,6 +54,7 @@ export async function TemplateRenderer( {
 				</main>
 			</TemplateScripts>
 			{ /* Hot Fix for adding classes to the body outside the root layout */ }
```

Comment:
Add a todo to check if we can patch this with either the portal or the `@wordpress/hooks` implementation.

Reply:
Whats this hot fix comment refer to?